package.json

```diff
@@ -11,13 +11,12 @@
     "lib"
   ],
   "dependencies": {
```

Comment:
I've updated some of the dependencies and removed `yosay` since it wasn't being used.